src/ui/game_window.c

```diff
@@ -4,8 +4,8 @@ int count = 0;
 
 bool pGameQuit = false;
 
-SDL_Rect imgMazeBlueCoins = {200, 3, 168, 216};
-SDL_Rect imgMazeBlueCoinsZoom = {4, 4, 672, 864};
+SDL_Rect imgMazeBlueWithoutCoins = {370, 4, 166, 215};
+SDL_Rect imgMazeBlueWithoutCoinsZoom = {0, 0, WINDOW_SCREEN_WIDTH, WINDOW_SCREEN_HEIGHT};
```

Comment:
Maybe we don't need to precise "without coins" if we only use this version

Reply:
[cb9f364](https://github.com/Livl-Corporation/livl-pacman/pull/16/commits/cb9f364c24b6463c2294891bd3b416bc2f62efc3)